### ISSUE TITLE
TypeError: 'tuple' object does not support item assignment

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla-cluster-tests#457]
We actually checked type of p[2], but still touched strange TypeError.

```
00:08:05 Exception in wrapped method setUp
00:08:05 Traceback (most recent call last):
00:08:05   File "/jenkins/slave/workspace/scylla-master-longevity-gce/label/gce3-qavpc/sdcm/tester.py", line 130, in wrapper
00:08:05     return method(*args, **kwargs)
00:08:05   File "/jenkins/slave/workspace/scylla-master-longevity-gce/label/gce3-qavpc/sdcm/tester.py", line 256, in setUp
00:08:05     self.create_test_stats()
00:08:05   File "/jenkins/slave/workspace/scylla-master-longevity-gce/label/gce3-qavpc/sdcm/tester.py", line 1132, in create_test_stats
00:08:05     self.stats['setup_details'] = self.get_setup_details()
00:08:05   File "/jenkins/slave/workspace/scylla-master-longevity-gce/label/gce3-qavpc/sdcm/tester.py", line 1034, in get_setup_details
00:08:05     p[2] = re.sub('\s', '_', p[2])
00:08:05 TypeError: 'tuple' object does not support item assignment
00:08:05 Cleaning up resources used in the test
```

```
    def get_setup_details(self):
        setup_details = {}
        is_gce = False
        for p in self.params.iteritems():
            if ('/run/backends/gce', 'cluster_backend', 'gce') == p:
                is_gce = True
        for p in self.params.iteritems():
            if p[1] in ['stress_cmd', 'stress_modes']:
                continue
            else:
                if is_gce and (p[0], p[1]) in \
                        [('/run', 'instance_type_loader'),
                         ('/run', 'instance_type_monitor'),
                         ('/run/databases/scylla', 'instance_type_db')]:
                    # exclude these params from gce run
                    continue
                elif p[1] == 'n_db_nodes' and isinstance(p[2], str) and re.search('\s', p[2]):  # multidc
                    p[2] = re.sub('\s', '_', p[2])
                setup_details[p[1]] = p[2]
```

@larisau can you help to look at it?

**Comment by larisau** [comment]
How n_db_nodes defined in the test yaml?

**Comment by amoskong** [comment]
*n_db_nodes: 4 4*


I changed it to ->   *n_db_nodes: '4 4'*


The problem should be solved.

**Comment by larisau** [comment]
thanks, investigating

**Comment by larisau** [comment]
fix: [#458](https://github.com/scylladb/scylla-cluster-tests/pull/458)

**Comment by larisau** [comment]
fixed by: 
https://github.com/scylladb/scylla-cluster-tests/pull/458
